sarahricker/thinkery
# Nojira mobile menu

## JIRA Ticket

nojira

## Description

A few sentences describing the overall goals of the pull request's commits.

We didn't have a design for the mobile menu, so I created one that's a little more modern and on brand to match the designs we have. 

Also expanded PR to also include solutions for an animated header search bar and several other spacing issues throughout header + menus both on desktop + mobile.

## Impacted Areas in Application

List general components of the site that this PR will affect:

* mobile menu (slide out menu + general alignment)
* desktop menu (search + general alignment)
* site-container (to allow for slideout menu)

## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

## Steps to Test or Reproduce

1. `git fetch origin`
2. `git checkout nojira-mobile-menu`
3. In Menu settings, select a menu + check box for Mobile menu location (I'm using primary menu as a tester locally). 
4. Visit site on desktop + test new animated search bar
5. Inspect site in responsive viewport + adjust screen width until mobile menu toggle is present. 
6. Toggle menu on and off at different viewports + see a nice looking mobile menu

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/yB22VAMAY8XNS/giphy.gif)
